test/inductor/test_torchinductor.py

```diff
@@ -5614,6 +5615,7 @@ def fn(in_ptr0, in_ptr1, in_ptr2):
             ),
         )
 
+    @skipIfRocm
```

Comment:
Is there a problem with torchvision compatibility version installed?

Reply:
It seems that when this test is running, it tries to establish a console. That's why, I skipped it.